src/main/java/org/embulk/input/zendesk/utils/ZendeskUtils.java

```diff
@@ -0,0 +1,167 @@
+package org.embulk.input.zendesk.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.base.Charsets;
+import org.embulk.input.zendesk.models.Target;
+
+import org.embulk.spi.Column;
+import org.embulk.spi.ColumnVisitor;
+
+import org.embulk.spi.Exec;
+import org.embulk.spi.PageBuilder;
+import org.embulk.spi.Schema;
+import org.embulk.spi.json.JsonParser;
+import org.embulk.spi.time.Timestamp;
+import org.slf4j.Logger;
+
+import java.util.Base64;
+
+public class ZendeskUtils
+{
+    private static final Logger logger = Exec.getLogger(ZendeskUtils.class);
+
+    private ZendeskUtils()
+    {}
+
+    public static boolean isSupportIncremental(final Target target)
+    {
+        return !Target.TICKET_FORMS.equals(target)
+                && !Target.TICKET_FIELDS.equals(target);
+    }
+
+    public static String convertBase64(final String text)
+    {
+        return Base64.getEncoder().encodeToString(text.getBytes(Charsets.UTF_8));
+    }
+
+    public static int numberToSplitWithHintingInTask(int count)
+    {
+        return count % ZendeskConstants.Misc.RECORDS_SIZE_PER_PAGE == 0
```

Comment:
better to use `Math.ceil`? 
from JavaDoc: 
```
Returns the smallest (closest to negative infinity) double value that is greater than or equal to the argument and is equal to a mathematical integer.
```

Reply:
fixed